internal/stanza/converter.go

```diff
@@ -55,6 +55,24 @@ func Convert(obsLog *entry.Entry) pdata.Logs {
 
 	insertToAttributeVal(obsLog.Body, lr.Body())
 
+	if obsLog.TraceId != nil {
+		var buffer [16]byte
+		copy(buffer[0:16], obsLog.TraceId)
+		lr.SetTraceID(pdata.NewTraceID(buffer))
+	}
+	if obsLog.SpanId != nil {
+		var buffer [8]byte
+		copy(buffer[0:8], obsLog.SpanId)
+		lr.SetSpanID(pdata.NewSpanID(buffer))
+	}
+	if obsLog.TraceFlags != nil {
+		flags := lr.Flags()
+		flags = flags & 0xFFFFFF00
```

Comment:
Unclear why the 8 LSB are masked.

Reply:
@alexvanboxel can you clarify this / add a comment?

Reply:
I've added a commit with a comment, but this comes from the logs.proto:

``` Flags, a bit field. 8 least significant bits are the trace flags as
defined in W3C Trace Context specification. 24 most significant bits are reserved
and must be set to 0. Readers must not assume that 24 most significant bits
will be zero and must correctly mask the bits when reading 8-bit trace flag (use
flags & TRACE_FLAGS_MASK). [Optional].```

See: https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto